app/models/tokite/hook_event/pull_request_review.rb

```diff
@@ -14,7 +14,7 @@ def fields
       def notify?
         return false unless hook_params[:action] == "submitted"
         if hook_params[:review][:state] == "commented"
-          hook_params[:review][:body]
+          hook_params[:review][:body] || ""
```

Comment:
`notify?` should return true or false because it's predicate. Moreover, `hook_params[:review][:body] || ""` is always evaluated to truthy value.